lib/rules/assertion-before-screenshot.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * @fileoverview Assert on the page state before taking a screenshot, so the screenshot is consistent
+ * @author Luke Page
+ */
+
+'use strict'
+
+const assertionCommands = [
+  // assertions
+  'should',
+  'and',
+  'contains',
+
+  // retries until it gets something
+  'get',
+
+  // not an assertion, but unlikely to require waiting for render
+  'scrollIntoView',
+  'scrollTo',
+];
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Assert on the page state before taking a screenshot, so the screenshot is consistent',
+      category: 'Possible Errors',
+      recommended: true,
```

Comment:
Should this be `recommended: true`? It's not being added to `cypress/recommended`, so it seems like this shouldn't be here.